### PR TITLE
Added Bands Of Mourning

### PR DESCRIPTION
**Pull request by SeanSWatkins**
A couple friends said the beginning was a bit slow, I didn't find that at all. This book really opened up Scadrial, showing more than the few areas we are exposed to in the original trilogy and the current Wax and Wayne. Also, creates more questions than answers as is the Sanderson way. There is some terrific character development too, we are introduced to a bunch of new, interesting characters and go in depth with a few already established ones.  Very satisfying novel.

### CODE REVIEW

**Comment by RichardLitt** [comment]
Have you read any of the others? Mind adding a comment above the list?

**Comment by SeanSWatkins** [comment]
I have read all of them, will add something

**Comment by RichardLitt** [comment]
@SeanSWatkins Ping!

**Comment by SeanSWatkins** [comment]
@RichardLitt Apologies! Completely slipped my mind. I have added a comment. Let me know if you want me to reduce it or edit another way

File: README.md [hunk 1]
```diff
@@ -189,6 +189,13 @@ The Mistborn trilogy (well, he plans on doing a trilogy of trilogies, and the la
...
+
+The first trilogy introduces us to a strange and spectacular world: one where there is magic, intrigue, social/economic inequality and, possibly my favorite thing, different races of people that inhabit this world.
+
+The second trilogy takes place some 300 years after the events of the first, and many of these events have filtered through into the 'modern' day Scadrial (it has a very old western feel to the whole setting); such as cities named after hero's, religions based on characters and many little secrets that are still unanswered from the first trilogy. We are introduced to a whole batch of new characters and some not so new characters (I won't say more otherwise I may give too much away).
+Would recommend this entire series and the whole Cosmere to any fantasy fan. [@SeanSWatkins](https://github.com/SeanSWatkins)
```
Comment (RichardLitt) [inline README.md]: Needs a dash between the period and the bracket. 

Otherwise, good!

Comment (SeanSWatkins) [inline README.md]: Should I just use a dash instead of parenthesis?

**Comment by SeanSWatkins** [comment]
Made a few fixes, lemme know if they are problematic

**Comment by RichardLitt** [comment]
Merged. Thanks!

**Pull request merged**